lib/cmds/login.js

```diff
@@ -38,9 +40,15 @@ export async function login () {
     return
   }
 
-  await opn(oAuthURL, {
-    wait: false
-  })
+  if (['win32', 'darwin'].includes(process.platform)) {
+    await opn(oAuthURL, {
+      wait: false
+    })
+  } else if (await commandExists('xdg-open')) {
+    await execa('xdg-open', [oAuthURL])
```

Comment:
is there any chance that this might throw an exception even if the command exists? Maybe we should `try` `catch` it